### ISSUE TITLE
WCAG 2.1 needed on the http://r4ds.had.co.nz   

### ISSUE THREAD

**Issue by WayneEDick** [hadley/r4ds#892]
This does not reflow correctly at 400%. I am fortunate to have a Kindle edition that does enlarge, but the webpage would be good. Use responsive design with media queries to restructure at 320 CSS Pixels at the very least. I really think restructuring should occur at multiple CSS pixel widths, maybe four cases. Also remember, large print on a laptop/desktop fails if stick headers or footers obscure the text. The best solution would be to make the TOC expandable and keep the header small.
Thank You, Wayne Dick
The following links give the reasons for this request:
https://www.w3.org/WAI/WCAG21/Understanding/reflow.html
For theory
https://assets.knowbility.org/training/2020/low-vision/Usability-of-Enlargement-Methods-Wayne-Dick.pdf

**Comment by hadley** [comment]
I've just updated the book to a new theme, which should have much better accessibility.
